lib/java/com/google/android/material/navigation/NavigationBarItemView.java

```diff
@@ -619,13 +619,15 @@ public void setEnabled(boolean enabled) {
     smallLabel.setEnabled(enabled);
     largeLabel.setEnabled(enabled);
     icon.setEnabled(enabled);
+  }
 
-    if (enabled) {
-      ViewCompat.setPointerIcon(
-          this, PointerIconCompat.getSystemIcon(getContext(), PointerIconCompat.TYPE_HAND));
-    } else {
-      ViewCompat.setPointerIcon(this, null);
+  @RequiresApi(api = VERSION_CODES.N)
```

Comment:
What about API < 24?

Can you explain why the original approach doesn't work?

Reply:
> Can you explain why the original approach doesn't work?

Original approach overwrites custom pointer icon (if set). Also it doesn't work correctly if itemView is enabled, but non-clickable.

> What about API < 24?

The pointer icon won't change, as it does when using ViewCompat.